docs/algorithms/ProjectionErrors.asciidoc

```diff
@@ -69,9 +69,9 @@ Below are a number of examples showing the error results with the Sinusoidal pro
 
 Below the Sinusoidal test case is used as an example showing how the distance and angle errors manifest.
  
-[[ProjectionErrorsSinusoidal1]]
+[#ProjectionErrorsSinusoidal1,reftext='{figure-caption} {counter:refnum}']
```

Comment:
This change from `[[ProjectionErrorsSinusoidal1]]` to `[#ProjectionErrorsSinusoidal1...]` fails to build the correct XML that is built into the PDF files.  While not completely familiar with the change that you propose here, the following does build the documents successfully.
```
[[ProjectionErrorsSinusoidal1,reftext='{figure-caption} {counter:refnum}']]
```